### PR TITLE
fix: reinstate Xcode 15 support

### PR DESCRIPTION
**Pull request by Brett-Best**

### CODE REVIEW

**Comment by Brett-Best** [comment]
@a7ex if you could review this please 🙏🏻

**Comment by Brett-Best** [comment]
In the meantime I've made a prerelease tag: https://github.com/a7ex/xcresultparser/releases/tag/1.7.1-prerelease-2024-07-30

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 5.10
```
Comment (maikelvdh) [inline Package.swift:1]: Shouldnt this rather be 5.9 to have largest compatibility with Xcode 15 versions? Specially given minimum supported Xcode version by brew is Xcode 15.0 which is having Swift 5.9, while Swift 5.10 is reaching only Xcode versions >= 15.3

Comment (maikelvdh) [reply Package.swift:1]: Raised https://github.com/a7ex/xcresultparser/pull/43 to address that.

**Pull request merged**